examples/http-client-oauth2-jwt-bearer-grant-type/http_client_oauth2_jwt_bearer_grant_type.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - OAuth2 JWT bearer grant type
 
-A client, which is secured with an OAuth2 JWT bearer grant type can be used to connect to a secured service. The client is enriched with the `Authorization: Bearer <token>` header by passing the `http:OAuth2JwtBearerGrantConfig` to the `auth` configuration of the client.
+An `http:Client` can be secured with OAuth2 JWT bearer grant type authorization. This is achieved by enriching each request from the client with the `Authorization: Bearer <token>` header. The required configurations for the OAuth2 JWT bearer grant type can be specified in the `auth` field as the `http:OAuth2JwtBearerGrantConfig` record. Use this to communicate with the service, which is secured with OAuth2 authorization.
```

Comment:
**When to use ?**
use when the client wants to receive access tokens without transmitting sensitive information such as the client secret

Reply:
Added via [187d99d](https://github.com/ballerina-platform/ballerina-distribution/pull/3793/commits/187d99dffcd52f92e019e05f224ac4dc5e4a2e66)